polar-core/src/vm.rs

```diff
@@ -963,7 +1010,7 @@ impl PolarVirtualMachine {
 
     /// Halt the VM by clearing all goals and choices.
     fn halt(&mut self) -> QueryEvent {
-        self.log("HALT", &[]);
+        self.log_with(LogLevel::Trace, || "HALT", &[]);
```

Comment:
Same question as above — this log is being moved from `Info` -> `Trace`. Is that intentional?